wfe2/wfe.go

```diff
@@ -1668,6 +1683,22 @@ func (wfe *WebFrontEndImpl) KeyRollover(
 	// Update the account key to the new key
 	updatedAcct, err := wfe.RA.UpdateRegistration(ctx, *acct, core.Registration{Key: &newKey})
 	if err != nil {
+		if berrors.Is(err, berrors.Duplicate) {
+			// It is possible that between checking for the existing key, and preforming the update
```

Comment:
For the "duplicate" case of key rollover, I don't think the spec requires us to return a Location header showing the preexisting account. Since it's possible to get that info with the new-account / onlyReturnExisting functionality, I'd rather do a simple error return here.

Reply:
This is specified, but not required:

> If there is an existing account with the new key
   provided, then the server SHOULD use status code 409 (Conflict) and
   provide the URL of that account in the Location header field.

Reply:
Nice, I had missed that. Thanks for the reference. Based on that I'm good keeping this.